config/locales/forms/es.yml

```diff
@@ -3,15 +3,14 @@ es:
   forms:
     backup_code:
       are_you_sure_desc_html:
-        - <strong>Los códigos de respaldo no son muy seguros</strong> porque se
-          pueden perder o robar fácilmente. <strong>Si peude, elija otra
-          opción</strong>, (usar un teléfono, descargar una aplicación de
-          autenticación o usar una clave de seguridad).
-        - <strong>Si continúa con los códigos de copia de seguridad, manténgalos
-          seguros</strong>. Le daremos 10 códigos que puede descargar, imprimir,
-          copiar o escribir. Más tarde, deberás ingresar un codigo cada vez que
-          inicies sesión.
-      are_you_sure_title: '¿Estás seguro? Le daremos códigos de respaldo para guardar y usar.'
+        - <strong>Los códigos de respaldo son el método de autenticación menos
+          utilizado,</strong> ya que se pueden perder con facilidad. Le
+          recomendamos explorar otras opciones de autenticación, como el
+          desbloqueo facial o táctil, una aplicación de autenticación o una
+          clave de seguridad.
+        - Le otorgaremos 10 códigos que puede descargar, imprimir, copiar o
+          escribir. Ingresará un código cada vez que inicie sesión.
+      are_you_sure_title: '¿Está seguro de que desea usar códigos de respaldo?'
```

Comment:
Spanish appears to be the only language with single quotes around the text, maybe the quotes should be removed?

Reply:
That is, for the key `are_you_sure_title`

Reply:
Good question. That field was in quotes before, as are many in the es yml. I assumed it was because of the special characters 🤷

Reply:
I believe for Spanish text, any kind of punctuation has to be escaped because it does not render the text as expected. I ran into this one, but I do not remember where...